app/tests/spec/lib/environment.js

```diff
@@ -97,6 +97,12 @@ define([
         windowMock.name = 'remote';
         assert.isFalse(environment.isFramed());
       });
+
+      it('returns `false` if the window\'s name is `payflow`', function () {
+        windowMock.top = new WindowMock();
+        windowMock.name = 'payflow';
```

Comment:
Add a line comment to issue #2607 here (or on line 100) for future versions of ourselves to remember why we're hardcoding a window name in this test?